data/human/hails.txt

```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
 		"uranium or uranium" "uranium"
 		"neodymium and neodymium" "neodymium"
 		"neodymium or neodymium" "neodymium"
-
+phrase "friendly civilian"
+    word
+        "Be careful of"
+        "A warning about"
+        "Be suspicious of"
+        "Don't trust"
+    word
+        " "
+    word
+        "those space casinos. I blew"
+    word
+        " "
+    word
+        "a"
+        "two"
+        "three"
+        "four"
+        "five"
+    word
+        " "
+    word
+        "million credits on"
+    word
+        "Bourne"
+        "Spica"
+        "Rust"
+        "Starcross"
+        "Skymoot"
+        "Thunder"
+        "Zug"
+        "Flood"
+        "Longjump"
+        "Glaze"
+        "Deep"
+        "Clink"
+        "Winter"
+        "Solace"
+        "Twinstar"
+        "Sundrinker"
+        "Trinket"
+        "Dancer"
```

Comment:
How come these are all Southern Rim planets? I feel like you'd probably find more gambling in the Paradise Worlds or the Core planets.